src/language/en.tsx

```diff
@@ -1,7 +1,10 @@
 const en = {
     languageName: 'English',
 
-    valuesHeaderEvaluation: 'Values',
+    navigationLabelActivites: 'Registation',    // Unused
+    navigationLabelCalendar: 'Calendar',        // Unused
+    navigationLabelValues: 'Values',            // Unused
+    navigationLabelInformation: 'Information',  // Unused
```

Comment:
If these are unused, shouldn't they be removed?

Reply:
We wasen't sure if we wanted to add them later so we keept them and if we want them, we only need to change a bool